src/StoreFileSystem.cc

```diff
@@ -70,6 +70,17 @@ StoreFileSystem::FreeAllFs()
     }
 }
 
+StoreFileSystem *
+StoreFileSystem::FindByType(const char *type)
+{
+    for (size_t i = 0; i < FileSystems().size(); ++i) {
+        auto fs = FileSystems().at(i);
```

Comment:
The callers do change `*fs` (fixing that is out of scope) but this code does not change the `fs` pointer.

```suggestion
        const auto fs = FileSystems().at(i);
```

Reply:
Done.